packages/web/src/components/settings-sheet.tsx

```diff
@@ -113,18 +113,17 @@ function PackageJson({ openDepsInstallModal }: { openDepsInstallModal: () => voi
         title="package.json"
         className={cn({ 'border-run': installing })}
       >
-        <div className="pt-1 pb-3 px-3">
-          <CodeMirror
-            value={source}
-            theme={codeTheme}
-            extensions={[
-              json(),
-              Prec.highest(keymap.of([{ key: 'Mod-Enter', run: evaluateModEnter }])),
-            ]}
-            onChange={onChangeSource}
-            basicSetup={{ lineNumbers: false, foldGutter: false }}
-          />
-        </div>
+        <CodeMirror
```

Comment:
I'm 50/50 on the readability improvement on this one. A surrounding div with only styles is easier to see than one of many props. Don't feel super strongly.